### ISSUE TITLE
Source removes sent for non-existent sources

### ISSUE THREAD

**Issue by legastero** [otalk/mod_muc_focus#41]
Needs to check that a source was actually present, or else an empty source-remove is sent.

**Comment by fippo** [comment]
which might have bad effects on the client.

**Comment by fippo** [comment]
```
<iq id='lx252' type='set' to='e17fd78d-0b44-4d72-aad5-ead97efae786@beta-im.talky.io/35ec52c2-d605-4551-8957-1e16f0e24479' from='update@beta-conference.talky.io'><jingle sid='1b7d5fce9a1a6015' initiator='update@beta-conference.talky.io' action='source-remove' xmlns='urn:xmpp:jingle:1'/></iq>`
```
I think that can happen when someone leaves before sending their ssrcs but it can also happen more often in the future with listen-only participants.

**Comment by fippo** [comment]
fixed in 6bc28b86b9a50f68eb9351a05f5d9bfc3af44262
